src/components/Footer/AyaFooter.js

```diff
@@ -7,13 +7,14 @@ export default class Footer extends Component {
   }
   render() {
     const styles = require('./AyaFooter.scss');
+    const currentYear = new Date().getFullYear();
     return (
     <div>
     <hr />
      <footer className={styles.footer}>
         <div className="container">
             <p className={styles.quote}>“The greatness of a nation and its moral progress can be judged by the way its animals are treated.” <span className="text-muted">Mahatma Gandhi</span></p>
-            <div className="row">
+            {/* <div className="row">
```

Comment:
Can you remove this if not neccessary

Reply:
I would like to keep this for the future, when I get these accreditations.